### ISSUE TITLE
Locales - French capitalization

### ISSUE THREAD

**Issue by zubalea** [amcharts/amcharts4#4085]
Hello,

In french, the month is capitalised only if it's at the beginning of a sentence.  
For example:   Mai (May) 

However when it has a day indicator before it, then it needs to be in lowercase.
For example:  7 mai 

Is there a way to handle the two different situation with the amchart locales?   

Thank you

**Comment by martynasma** [comment]
No, I'm afraid. We do not have separate translations for when it is used in beginning of the sentence and not :(
